docs/guide/running-extern-robot-controllers.md

```diff
@@ -121,6 +135,13 @@ java -classpath $WEBOTS_HOME/lib/controller/java/Controller.jar:$WEBOTS_HOME/pro
 
 &nbsp;
 
+**Python**: Add the correct python version:
+
+When setting or extending the `PYTHONPATH` environment variable, make sure to replace the `X` in `python3X` with your python version.
+This can be found by typing `python3 --version` in the terminal. If for example the answer is `Python 3.8.10`, `python3X` should be `python38`.
```

Comment:
```suggestion
This can be found by typing `python3 --version` in the terminal.
If for example the answer is `Python 3.8.10`, `python3X` should be `python38`.
```